packages/react/src/components/Stack/Stack.styles.ts

```diff
@@ -7,22 +7,24 @@ const nameMap: { [key: string]: string } = {
   end: 'flex-end',
 };
 
-const GlobalClassNames = {
+export const GlobalClassNames = {
   root: 'ms-Stack',
   inner: 'ms-Stack-inner',
+  child: 'ms-Stack-child',
```

Comment:
Should `ms-StackItem` be added here as well?

Reply:
`ms-StackItem` is added in `StackItem.styles.ts`. I'll export it from there and import it here.